swagger.yaml

```diff
@@ -240,14 +244,16 @@ paths:
           type: "integer"
           format: "int64"
       responses:
-        204:
+        200:
```

Comment:
Just curious, why did you change this from a 204? I would say a 204 is more common when deleting a resource, but either are acceptable.